pos_membership/__openerp__.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010-2015 Elico Corp (<http://www.elico-corp.com>)
+#    Alex Duan <alex.duan@elico-corp.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{'name': 'Membership Management - POS Membership',
+ 'version': '1.0',
+ 'category': 'Generic Modules',
+ 'depends': [
+     'membership_account_balance',
+     'point_of_sale',
+     'account_accountant',
+     'account_voucher'
+ ],
+ 'author': 'Elico Corp',
+ 'license': 'AGPL-3',
+ 'website': 'https://www.elico-corp.com',
+ 'description': """
```

Comment:
yes: it should appear only in README. Please read carefully the 
guidelines here below:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3


README:
Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst